atest/testdata/test_libraries/LibUsingLoggingApi.py

```diff
@@ -24,9 +24,17 @@ def write_messages_to_console():
     logger.console('in two parts')
     logger.info('To log and console', also_console=True)
 
+
 def log_non_strings():
     logger.info(42)
     logger.warn(True)
 
+
 def log_callable():
     logger.info(log_callable)
+
+
+def log_error_with_print():
+    print "*ERROR* error msg with print"
```

Comment:
As already commented, this suite/library is about `robot.api.logger`, not about logging using `print`.